doc/2/guides/migrate-from-v1/introduction/index.md

```diff
@@ -0,0 +1,21 @@
+---
+code: false
+type: branch
+title: Migration Guide
+order: 100
+---
+
+# Migrate from Kuzzle v1 to Kuzzle v2
+
+## Breaking changes
+
+  - End of Node.js 6 support
+  - End of Redis 3 and 4 support
+  - End of Elasticsearch 5 support
+
+## New Features
+
+  - Support for Node.js 10
+  - Support for Elasticsearch 7
+
+## Enhancements
```

Comment:
If there is no enhancements why keeping this part?
```suggestion
```

Reply:
There may be enhancements later

Reply:
Shouldn't we just list breaking changes, and let the changelog generator take care of the rest when we release?